src/components/Dialog-Window/Resources/DialogResources.module.scss

```
@@ -0,0 +1,19 @@
+.li,
+a {
+  color: rgba(31, 103, 199, 1);
+}
+
+.input {
+  width: 14rem;
```

Hard coded widths should be avoided, but this will _probably_ be ok also on small phones.
```suggestion
  // TODO: Add `display: flex` to container and switch out the hard coded `width` for `flex-grow: 1`
  width: 14rem;
```

---

Is this todo for another pr or?

---

You decide. It's such a small change that I'd make it part of this one.

---

Fixed width by `flex-grow: 0`?